### ISSUE TITLE
[Feature] Tag by Job

### ISSUE THREAD

**Issue by rafa-acioly** [jobrunr/jobrunr#595]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
The possibility to add `Tag` to each job, currently we can set a `Tag` on the server but not on the job, It would be useful to have a tag on each job to separate it better, this would give us more visibility on how much job is being generated by "tag".

**Comment by rdehuyss** [comment]
Hi @rafa-acioly !! I'm not completely following: are you meaning the Tags from JobRunr Pro?

**Comment by rafa-acioly** [comment]
@rdehuyss No, JobRunr Pro has Tags to tag the server, what I meant was a tag per job, e.g:

I've 3 different types of jobs, one to sync the product information, one to sync images and another one to sync prices (there are not related or depend on each other)

I would like to know by looking at the dashboard what each job is related to.

**Comment by rdehuyss** [comment]
I still don't understand your feature request to be honest. What about the display name? That will be the same for each type of job?

**Comment by rdehuyss** [comment]
Feedback from @rafa-acioly:

I was wondering if it's possible to tag / label each job, I've many numbers of different tasks and I want to see on the dashboard that each job is related to.

**Comment by rdehuyss** [comment]
Solved in JobRunr v6.
